### ISSUE TITLE
PageNav in CatView

### ISSUE THREAD

**Issue by mambax7** [XoopsModules25x/wfdownloads#28]
However, i noticed that there is a problem whith the "pagenav" class.
For example the link of the second page of a given category (when we set 5 files per page), instead of being like this : "/modules/wfdownloads/viewcat.php?cid=1&start=6", i have this : "/modules/wfdownloads/viewcat.php?start=5&list=".

**Comment by mambax7** [comment]
fixed in: https://github.com/XoopsModules25x/wfdownloads/pull/27/commits/1927dde06e34db9186d553fd45a11a37c62977bc

**Comment by Aminov81** [comment]
Problem solved indeed.
If you allow me to make a proposition. I think it would be better to build the URLs of navigation pages like this : "/modules/wfdownloads/viewcat.php?cid=1&start=6". I mean to put the parameter "cid" before the parameter "start". It will make the URL Rewriting much easier (to have this for example as result : "/documents/category-1/start=6").
An other thing, when we set "5" downloads per page, the second page of navigation starts at "6" and not at"5". The third page starts at "11" and not at "10". Generally speaking, when we set "n" downloads per page, the second page starts with "n+1".

**Comment by mambax7** [comment]
The array starts at 0, so that why we have it that way. I would have to look into it, but right now it is more as an enhancement than a bug

**Comment by Aminov81** [comment]
Yes sure, it's not a bug of course, i just took the oportunity to make a proposition of enhancement as you said :-)

**Comment by mambax7** [comment]
Of course, and I appreciate it! Thank you!
